shared/src/packages/trucks/libs/validation-schemas/truck-create-request-body.validation-schema.ts

```diff
@@ -0,0 +1,63 @@
+import joi from 'joi';
+
+import { FormLabel, TruckManufacturer, TruckTowType } from '../enums/enums.js';
+import { type TruckEntity } from '../types/types.js';
+
+const truckCreateRequestBodyValidationSchema = joi.object<
+  Omit<TruckEntity, 'id'>,
+  true
+>({
+  manufacturer: joi
+    .string()
+    .valid(...Object.values(TruckManufacturer))
+    .messages({
+      'any.only': `Invalid ${FormLabel.MANUFACTURER}`,
+      'any.required': `${FormLabel.MANUFACTURER} is required`,
+    }),
+
+  capacity: joi
+    .number()
+    .min(0)
+    .required()
+    .messages({
+      'number.base': `${FormLabel.CAPACITY} must be a number`,
+      'number.min': `${FormLabel.CAPACITY} must be at least 0`,
+    }),
+
+  pricePerKm: joi
+    .number()
+    .precision(1)
+    .min(1)
+    .max(100)
```

Comment:
and here you must use your constants too.

Also, why didn't you make a separate templates to reduce boiler-code. Currently i can see this rule in two places.